docs/en/getting-started/example-datasets/star-schema.md

```diff
@@ -1,36 +1,36 @@
 ---
 slug: /en/getting-started/example-datasets/star-schema
 sidebar_label: Star Schema Benchmark
-description:  "Dataset based on the TPC-H dbgen source. The coding style and architecture follows the TPCH dbgen."
+description:  "The Star Schema Benchmark (SSB) data set and queries"
 ---
 
 # Star Schema Benchmark (SSB, 2009)
 
-The Star Schema Benchmark is roughly based on the TPC-H tables and queries but unlike TPC-H, it uses a star schema layout.
+The Star Schema Benchmark is roughly based on the [TPC-H](tpch.md)'s tables and queries but unlike TPC-H, it uses a star schema layout.
 The bulk of the data sits in a gigantic fact table which is surrounded by multiple small dimension tables.
 The queries joined the fact table with one or more dimension tables to apply filter criteria, e.g. `MONTH = 'JANUARY'`.
 
 References:
-[Star Schema Benchmark](https://cs.umb.edu/~poneil/StarSchemaB.pdf) (O'Neil et. al), 2009
-[Variations of the Star Schema Benchmark to Test the Effects of Data Skew on Query Performance](https://doi.org/10.1145/2479871.2479927) (Rabl. et. al.), 2013
-
-
+- [Star Schema Benchmark](https://cs.umb.edu/~poneil/StarSchemaB.pdf) (O'Neil et. al), 2009
+- [Variations of the Star Schema Benchmark to Test the Effects of Data Skew on Query Performance](https://doi.org/10.1145/2479871.2479927) (Rabl. et. al.), 2013
 
 First, checkout the star schema benchmark repository and compile the data generator:
+
 ``` bash
-$ git clone https://github.com/vadimtk/ssb-dbgen.git
-$ cd ssb-dbgen
-$ make
+git clone https://github.com/vadimtk/ssb-dbgen.git
+cd ssb-dbgen
+make
 ```
 
-Then, generate the data. Parameter `-s` specifies the scale factor. For example, with `-s 100`, 600 million rows (67 GB) are generated.
+Then, generate the data. Parameter `-s` specifies the scale factor. For example, with `-s 100`, 600 million rows are generated.
 
 ``` bash
-$ ./dbgen -s 1000 -T c
-$ ./dbgen -s 1000 -T l
-$ ./dbgen -s 1000 -T p
-$ ./dbgen -s 1000 -T s
-$ ./dbgen -s 1000 -T d
+declare -a tables=("c" "l" "p" "s" "d")
```

Comment:
Also, the usage of double quotes " is not motivated (in contrast to single quotes).